public/data/anurag.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Anurag",
+    "bio": "Teen | Front-end Developer | Student",
+    "avatar": "https://github.com/kr-anurag.png",
+    "links": [
+      {
+        "name": "Checkout my Projects on Github",
+        "url": "https://github.com/kr-anurag",
+        "icon": "github"
+      },
+      {
+        "name": "Connect with me on Twitter",
+        "url": "https://twitter.com/kr_anurag_",
+        "icon": "twitter"
+      },
+      {
+          "name": "Join my Discord community",
+          "url": "https://discord.gg/MqsS6axMrZ",
+          "icon": "discord"
+      }
+    ]
+  }
```

Comment:
```suggestion
}

```

Reply:
apparently doing this won't solve the issue, what should I do now? like can I make a new PR?

Reply:
Sure, but best to this here itself.

Reply:
actually, there have been  9 commits now, by pulling and all that stuff, and I'm really sorry, I don't think I can solve it here itself, sorry! But I will make a new one

Reply:
Sure, feel free to close this PR and open a new one 😄